tests/e2e/e2e_test.go

```diff
@@ -220,15 +220,19 @@ func TestCloudControllerManagerLoadBalancersWithPrivateNetwork(t *testing.T) {
 }
 
 func TestCloudControllerManagerNetworksPodIPsAreAccessible(t *testing.T) {
-	if testCluster.useNetworks == false {
-		t.Skipf("Private Networks test is disabled")
+	node, err := testCluster.k8sClient.CoreV1().Nodes().Get(context.Background(), testCluster.scope+"-1", metav1.GetOptions{})
+	if err != nil {
+		t.Error(err)
 	}
 
-	nwTest := nwTestHelper{t: t, K8sClient: testCluster.k8sClient, privateKey: testCluster.setup.privKey, podName: "network-routes-accessible"}
-
-	pod := nwTest.DeployTestPod()
-
-	WaitForHTTPOnServer(t, testCluster.setup.ExtServer, testCluster.setup.privKey, pod.Status.PodIP, false)
-
-	nwTest.TearDown()
+	network, _, err := testCluster.hcloud.Network.Get(context.TODO(), testCluster.scope)
```

Comment:
Depending on the timing and execution order of the tests the hccm might not have setup the routes yet. To avoid any flaky tests we should poll for this condition.

Might be something for a follow up PR though.